index.html

```diff
@@ -3004,20 +3004,25 @@ <h2>
 DID Resolution
         </h2>
         <p>
-The <a>DID resolution</a> function resolves a <a>DID</a> into a <a>DID
+The <a>DID resolution</a> functions resolve a <a>DID</a> into a <a>DID
 document</a> by using the "Read" operation of the applicable <a>DID method</a>.
 (See <a href="#read-verify"></a>.) The details of how this process is
 accomplished are outside the scope of this specification, but all conformant
-implementations MUST implement a function which has the following abstract form:
+implementations MUST implement two functions which have the following abstract forms:
         </p>
 
         <p><code>
 resolve ( did, did-resolution-input-metadata ) <br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document, did-document-metadata )
+        </code></p>
+
+        <p><code>
+resolveStream ( did, did-resolution-input-metadata ) <br>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document-stream, did-document-metadata )
         </code></p>
 
         <p>
-The input variables of this function MUST be as follows:
+The input variables of these functions MUST be as follows:
```

Comment:
I would recommend splitting the definition of these functions into separate subsections with different descriptions about their inputs and outputs instead of trying to put everything in one block as they are here. This is possible to do as an editorial change after this PR is merged, but the issue shouldn't be lost.

Reply:
Agreed, let's do this in a separate PR given that this has taken so long to get into the spec. Let's set down the base, editorial changes will be easier after that's done.